SecurityCardManagement/README.md

```diff
@@ -36,10 +36,9 @@ Before forwarding the request to a security card, a browser must request a pin c
 arguments: 
 
 	answerId: uint32 - function id of result callback	
-	sn	: uint192 - card serial number
```

Comment:
Why removing it? You could use this info to bind the keys data to the card ID

Reply:
We can restore this, if really need it.